static/global.css

```diff
@@ -13,6 +13,7 @@
   --acm-red: #d41153;
   --acm-sky: #38b6ff;
   --acm-canvas: #eaeaea;
+  --acm-codeblock-hover: #eaeaea;
```

Comment:
--acm-canvas-hover maybe better name maybe not only use for code block

Reply:
or canvas-2, canvas-darker, etc?

Reply:
done, removed the color cus we're using button